app/serializers/search/listing_result_serializer.rb

```diff
@@ -0,0 +1,29 @@
+module Search
+  # TODO: [@atsmith813] Rename this to just ListingSerializer once Elasticsearch
+  # is fully removed
+  class ListingResultSerializer < ApplicationSerializer
+    attributes :id,
+               :body_markdown,
+               :bumped_at,
+               :category,
+               :contact_via_connect,
+               :expires_at,
+               :originally_published_at,
+               :location,
+               :processed_html,
+               :published,
+               :slug,
+               :title,
+               :user_id
+
+    attribute :tags do |cl|
+      cl.cached_tag_list.to_s.split(", ")
+    end
+
+    attribute :author do |cl|
+      ListingAuthorSerializer.new(cl.author)
```

Comment:
I suspect when this "transition" is complete we can refactor this serializer to something more generic that can be re-used.